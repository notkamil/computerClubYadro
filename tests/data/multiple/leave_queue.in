1
08:00 20:00
10
09:00 1 client2
10:00 1 client1
11:00 2 client1 1
12:00 3 client2
14:00 4 client2
15:00 4 client1
