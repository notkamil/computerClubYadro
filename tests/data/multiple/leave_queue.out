08:00
09:00 1 client2
10:00 1 client1
11:00 2 client1 1
12:00 3 client2
14:00 4 client2
15:00 4 client1
20:00
1 40 04:00